### ISSUE TITLE
Conversations 1.18.5 crashes on reentering conference you don't participate at

### ISSUE THREAD

**Issue by tdemin** [siacs/Conversations#2539]
Conversations will crash on opening the conference we don't participate at if we open a chat and switch to another, then going back to this conference.

The issue actually doesn't happen every time, but approximately with 60-70% chance.

#### Information

* **Version:** 1.18.5 from Google Play Store
* **Android Version:** 7.1.2
* **Server name:** [jabber.cat](https://jabber.cat), Prosody
* **Conference Conversations can't open:** [linuxforum@conference.feder8.ru](xmpp:linuxforum@conference.feder8.ru)
* **Server modules:** everything on the "Server Info" from the account screen is present

#### Steps to reproduce

1. Join a conference you don't participate at.
2. Open any other chat.
3. Open the conference tab again.
4. "Conversations has stopped".

#### Stack trace

```
java.lang.IllegalArgumentException: inputConnection must be non-null
        at android.support.v13.view.inputmethod.InputConnectionCompat.createWrapper(InputConnectionCompat.java:272)
        at eu.siacs.conversations.ui.EditMessage.onCreateInputConnection(EditMessage.java:154)
        at android.view.inputmethod.InputMethodManager.startInputInner(InputMethodManager.java:1200)
        at android.view.inputmethod.InputMethodManager.restartInput(InputMethodManager.java:1142)
        at android.widget.TextView.setText(TextView.java:4400)
        at android.widget.TextView.setText(TextView.java:4337)
        at android.widget.EditText.setText(EditText.java:89)
        at android.widget.TextView.setText(TextView.java:4312)
        at eu.siacs.conversations.ui.ConversationFragment.reInit(ConversationFragment.java:909)
        at eu.siacs.conversations.ui.ConversationActivity$1.onItemClick(ConversationActivity.java:218)
        at android.widget.AdapterView.performItemClick(AdapterView.java:310)
        at android.widget.AbsListView.performItemClick(AbsListView.java:1164)
        at android.widget.AbsListView$PerformClick.run(AbsListView.java:3139)
        at android.widget.AbsListView$3.run(AbsListView.java:4054)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6186)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

**Comment by iNPUTmice** [comment]
That's fixed in 1.19.0

**Comment by tdemin** [comment]
@iNPUTmice that's good, but I can't see Conversations 1.19.0 on Play Store. Anything's wrong?
